resume/resume.html

```diff
@@ -0,0 +1,95 @@
+            <p>David.sejna@seznam.cz</p>
+        </div>
+    </div>
+    <div>
+        <h1>Cíl</h1>
+        <p class="main">
+            Aktuálně studuji vysokou školu ekonomickou, konkrétně obor informační média a služby.
+            Dlouhodobě jsem se věnoval pracím mimo obor a mým aktuálním cílem je získat při studiu více praxe z IT
+            oborů.
+            Svět technologií jde dopředu a rád s ním šel také.
+        </p>
+        <h1>Vzdělání</h1>
+        <h2><em><strong>SMÍCHOVSKÁ STŘEDNÍ PRŮMYSLOVA ŠKOLA</strong></em></h2>
+        <h3>2009 &ndash; 2013</h3>
+        <p class="main">Vystudovaný obor &ndash; Informační technologie</p>
+        <p class="main"> Za studium této školy jsem vedl několik projektů, pro mě nejdůležitější byla maturitní práce,
+            při které jsme
+            natáčeli studentský film, kde bylo třeba organizovat velké
+            počty úkolů a komunikovat s mnoha členy týmu. Získal jsem také certifikát MOS (microsoft office specialist)
+            a v
+            rámci praxí jsem měl na starosti rekonstrukci počítačové
+            sítě v několika učebnách.</p>
+        <h1>Pracovní zkušenosti</h1>
+        <h2>Moderátor/Animátor | <em><strong>Aquapalace Praha</strong></em></h2>
+        <h3>03/2016 &ndash; dosud</h3>
+        <p class="main">Hlavní náplní práce je primárně vedení týmu a následná komunikace s koncovými klienty. Věnoval
+            jsem se
+            vedení
+            animačních programů a za úspěchy považuji moderování akce
+            Integrovaného záchranného systému. Dokázal jsem vést tým a komunikovat před desítkami lidí. Tato pozice mi
+            zlepšila mé komunikační schopnosti s klienty a vedení týmu.</p>
+        <h2>Pracovník expedice | <em><strong>CZC.CZ</strong></em></h2>
+        <h3>03/2016 &ndash; 09/2016</h3>
+        <p class="main">Měl jsem na starosti celkový chod expedičního oddělení. Na starost jsem měl jak samotnou
+            přípravu zásilek,
+            tak i
+            organizaci jednotlivých položek.
+            Pracoval jsem s interním systémem a spolupracoval s ostatními odděleními například skladem anebo
+            reklamačním
+            oddělením.</p>
+        <h1>Dovednosti</h1>
+        <ul class="left">
+            <li>Komunikace</li>
+            <li>Vedení týmu</li>
+            <li>Word, excel</li>
+        </ul>
+        <ul class="right">
+            <li>Znalost HW/SW</li>
+            <li>Angličtina – pokročilá znalost psaní a porozumění, hovorově domluva o běžných věcech</li>
+        </ul>
+        <h1>Aktivity</h1>
+        <p class="main"><em>Čtení internetových portálu technického směru (zive.cz, atd.), hudba a DJing – organizace
+                vlastních
+                akcí,
+                sport – posilování, běh – při škole jsem vytvořil skupinu a
+                snažil se sjednotit lidi, co nemají s kým běhat</em></p>
+        <p class="main">Poznámky:</p>
+    </div>
+</body>
+
+</html>
```

Comment:
veškeré tvé práce musí být v rámci tvé složky www/sejd00/ - nesmíš to dávat někam mimo ni.

úkol musí být odevzdán přes git commit + git push z tvé branche - nesmí se nahrát přes web github. Odevzdej to prosím opravený znova z IDE nebo z command line, ty dva pull requesty musím zamítnout.